stable_baselines3/common/buffers.py

```diff
@@ -179,7 +175,6 @@ def __init__(
         device: Union[th.device, str] = "auto",
         n_envs: int = 1,
         optimize_memory_usage: bool = False,
-        handle_timeout_termination: bool = True,
```

Comment:
actually, this should stay here, otherwise we cannot use `optimize_memory_usage` option